lib/fastlane/plugin/wpmreleasetoolkit/actions/common/comment_on_pr.rb

```diff
@@ -10,7 +10,10 @@ class CommentOnPrAction < Action
       def self.run(params)
         require_relative '../../helper/github_helper'
 
-        reuse_identifier = Fastlane::Helper::GithubHelper.comment_on_pr(
+        token = params[:access_token]
+        github_helper = Fastlane::Helper::GithubHelper.new(github_token: token)
+
+        reuse_identifier = github_helper.comment_on_pr(
```

Comment:
Self-Review: As this is the only place where the `github_token` is called `access_token` and is a de facto user of `GITHUB_TOKEN` env variable, I use the same convention for the other actions in the project.